c8y-lora-sdk/src/main/java/com/cumulocity/lpwan/codec/sdk/CodecMicroserviceApplication.java

```diff
@@ -0,0 +1,18 @@
+package com.cumulocity.lpwan.codec.sdk;
+
+import com.cumulocity.lpwan.codec.sdk.rest.CodecRestController;
+import com.cumulocity.lpwan.codec.sdk.service.CodecService;
+import com.cumulocity.microservice.autoconfigure.MicroserviceApplication;
+import org.springframework.context.annotation.Import;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.TYPE})
```

Comment:
- [ ] Add a comment to explain the purpose of this annotation and where the microservice implementer should be using it.